fastlane/Fastfile

```diff
@@ -57,6 +57,12 @@ ENV['PUBLIC_CONFIG_FILE'] = PUBLIC_CONFIG_FILE
 INTERNAL_CONFIG_FILE = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
 ENV['FASTLANE_WWDR_USE_HTTP1_AND_RETRIES'] = 'true'
 
+# Fastlane's `git_branch` action and it's relevant helpers use environment variables to modify the output.
```

Comment:
```suggestion
# Fastlane's `git_branch` action and its relevant helpers use environment variables to modify the output.
```

Looks like I made a spelling mistake in the [WPAndroid one](https://github.com/wordpress-mobile/WordPress-Android/commit/c649f32927691d1e75f0f163f1aa5646ec55a7ae) 🤦‍♂️  I addressed it in https://github.com/wordpress-mobile/WordPress-Android/pull/19781.

Reply:
🤦‍♂️ on me for not noticing either. 🙃